### PR TITLE
Animations pt1

### PR DESCRIPTION
**Pull request by amymc**
Here goes nothing 🙈 

https://github.com/viewstools/morph/issues/11

Still missing a couple of things:
1. Hover animations
2. Transitions on web

Handling animations of the same properties across multiple scopes (https://github.com/viewstools/morph/issues/11#issuecomment-373873989) is not covered here either.

### CODE REVIEW

File: morph/react-dom.js [hunk 1]
```diff
@@ -8,7 +8,6 @@ import toComponent from './react/to-component.js'
 import walk from './walk.js'
 
 const imports = {
-  Animated: "import Animated from 'react-dom-animated'",
```
Comment (dariocravero) [inline morph/react-dom.js]: shouldn't we keep this one in web too?

Comment (amymc) [reply morph/react-dom.js]: Tbh I wasn't sure about this, but it seemed like it's not being used anywhere?

Comment (dariocravero) [reply morph/react-dom.js]: aren't the springs in dom using animated too?

Comment (amymc) [reply morph/react-dom.js]: Correct, but not the same library. I'm using [this one](https://github.com/animatedjs/animated) instead. I'm adding the import statement for that in `get-dependencies` [here](https://github.com/viewstools/morph/blob/55ef7963fa2a75d6a82022457ab9b78f03d32a99/morph/react/get-dependencies.js#L77-L79)

Isn't `react-dom-animated` basically obsolete now?

Comment (dariocravero) [reply morph/react-dom.js]: missed that :) that's fine then, thanks!

**Comment by dariocravero** [comment]
this absolutely epic 🎉!!!! well done @amymc, you rock!!! 🤘thank you so much!!!

**Pull request merged**